site/templates/siteComponent.pug

```diff
@@ -144,6 +144,13 @@ html(lang='en-US' dir="ltr").spectrum.spectrum--light.spectrum--medium
                 h1.spectrum-CSSComponent-title.spectrum-Heading.spectrum-Heading--sizeXXXL.spectrum-Heading--serif
                   a(href=`#${component.slug}`).spectrum-BigSubtleLink #{component.name}
 
+              if component.status == "Deprecated"
+                .spectrum-InLineAlert.spectrum-InLineAlert--negative(style="max-inline-size: 600px")
+                  h3.spectrum-InLineAlert-header Deprecated component
+                    svg(class="spectrum-Icon spectrum-Icon--sizeM spectrum-InLineAlert-icon" focusable="false" aria-hidden="true" role="img" xmlns="http://www.w3.org/2000/svg" height="36" viewBox="0 0 36 36" width="36")
+                      path(d="M17.127 2.579.4 32.512A1 1 0 0 0 1.272 34h33.456a1 1 0 0 0 .872-1.488L18.873 2.579a1 1 0 0 0-1.746 0ZM20 29.5a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-3a.5.5 0 0 1 .5-.5h3a.5.5 0 0 1 .5.5Zm0-6a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-12a.5.5 0 0 1 .5-.5h3a.5.5 0 0 1 .5.5Z")
+                  p.spectrum-InLineAlert-content This component has been deprecated. Use an <a href="actionbar.html">action bar</a> to allow users to perform actions on either a single or multiple items at the same time, instead.
```

Comment:
Love having this in the template 🙌 We'll probably need to be a bit more general in this message, but could still include recommendations for comparable components in the Migration Guidelines section

Reply:
haha yes more general indeed! :woman-facepalming: silly me.